src/en/users-workflows.md

```diff
@@ -0,0 +1,94 @@
+Title: Juju workflow scenarios
+TODO: 
+
+
+# Workflow scenarios
+
+To demonstrate multi-user interaction, we present several workflows. It is
+assumed that Juju is installed [link to juju install page]. This presentation
+will be based on the local cloud: LXD. Since LXD does not require credentials,
+there will be no mention of credentials below (see
+[Getting Started](./getting-started.html) for a full treatment of using LXD as
+the cloud provider).
+
+
+## Basic setup and single user
+
+**Controller creation**<br/>
```

Comment:
I agree, but when the header formats don't suck, someone will have to come and change all these back again